Library/Formula/rust.rb

```diff
@@ -31,8 +31,11 @@ class Rust < Formula
     sha256 "04f68ef26b4ca636b35be1d17ad2b641727bbc10cec52582a9e3a64b5784dbeb" => :mavericks
   end
 
+  option "with-llvm", "Build with brewed LLVM"
```

Comment:
I have now uploaded a new commit that changes the option name back to `--with-llvm`, and keeps the more descriptive description.